core/crypto/crypto_test.go

```diff
@@ -264,6 +264,55 @@ func TestRegistrationSameEnrollIDDifferentRole(t *testing.T) {
 	}
 }
 
+func TestTLSCertificateDeletion(t *testing.T) {
+	conf := utils.NodeConfiguration{Type: "peer", Name: "peer"}
+
+	peer, err := registerAndReturnPeer(conf.Name, nil, conf.GetEnrollmentID(), conf.GetEnrollmentPWD())
```

Comment:
@christo4ferris 
As registerAndReturnPeer is a helper function being written in inside the test file and not part of the functionality I think it wouldn't be needed. Actually this helper could be removed if register (peer/client/validator) function just returned the registered peer, that change could be accomplished in a new PR as some refactoring would be needed. Does it make sense?